source/content/guides/drupal-9/drupal-9-hosted-createcustom/10-deploy-live.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Migrate a Site That Was Created with a Custom Upstream to Drupal 9
+subtitle: Deploy Live
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-createcustom/deploy-live
+anchorid: deploy-live
+editpath: drupal-9/drupal-9-hosted-createcustom/10-deploy-live.md
+---
+
+<Partial file="drupal-9/deploy-using-relaunch.md" />
```

Comment:
Again, relaunch process is not needed here. Just push to the main branch and for each site, push the updates to master, then deploy to staging and live and done!